sklearn/metrics/classification.py

```diff
@@ -498,8 +498,13 @@ def f1_score(y_true, y_pred, labels=None, pos_label=1, average='binary',
     y_pred : 1d array-like, or label indicator array / sparse matrix
         Estimated targets as returned by a classifier.
 
-    labels : array
-        Integer array of labels.
+    labels : array-like, optional
```

Comment:
I don't think "list-like" is used elsewhere in the codebase for lists of labels/classes... I'm fine for this to be `list`.